### ISSUE TITLE
Inconsistent GraphQL Operation Name casing in generators

### ISSUE THREAD

**Issue by dthyresson** [redwoodjs/redwood#2764]
Andrew Lam from the Discord Community asked:

> Hi, I'm curious what is the reason for using different casing for the name of query and mutation?

![image](https://user-images.githubusercontent.com/1051633/121186875-81b6e100-c835-11eb-95f0-46f45ca10792.png)

There is inconsistency in the operation name casing and redwood could standardize: 

* Pascal Case (OperationName)
* Capitalized Snake Name (OPERATION_NAME)

## Cell Generators

The generator sets the same `operationName` which is PascalCase:

### List

`ModelsQuery`

```
export const QUERY = gql`
  query ${operationName} {
    ${camelName} {
      id
    }
  }
`
```

### Single Item

`FindModelQuery`

```
export const QUERY = gql`
  query ${operationName}($id: ${idType}!) {
    ${camelName}: ${camelName}(id: $id) {
      id
    }
  }
`
```

### Scaffolds

`FIND_${singularConstantName}_BY_ID`
`Update${singularPascalName}Mutation`
`Delete${singularPascalName}Mutation`
`Create${singularPascalName}Mutation`

I think just `FIND_${singularConstantName}_BY_ID` should change to be PascalCase -- the reason being that the function instead is CAPITAL_SNAKE_CASE and the Operation Name is Pascal Case:

This is in EditNameCell and NameCell.

Various exmaples.

```
export const QUERY = gql`
  query FIND_${singularConstantName}_BY_ID($id: ${idType}!) {
    ${singularCamelName}: ${singularCamelName}(id: $id) {<% columns.forEach(column => { %>
      <%= column.name %><% }) %>
    }
  }
`

export const QUERY = gql`
  query FIND_${singularConstantName}_BY_ID($id: ${idType}!) {
    ${singularCamelName}: ${singularCamelName}(id: $id) {<% columns.forEach(column => { %>
      <%= column.name %><% }) %>
    }
  }

export const QUERY = gql`
  query ${pluralConstantName} {
    ${pluralCamelName} {<% columns.forEach(column => { %>
      <%= column.name %><% }) %>
    }
  }

export const QUERY = gql`
  query ${pluralConstantName} {
    ${pluralCamelName} {<% columns.forEach(column => { %>
      <%= column.name %><% }) %>
    }
  }
`
`
const UPDATE_${singularConstantName}_MUTATION = gql`
  mutation Update${singularPascalName}Mutation($id: ${idType}!, $input: Update${singularPascalName}Input!) {
    update${singularPascalName}(id: $id, input: $input) {<% columns.forEach(column => { %>
      <%= column.name %><% }) %>
    }
  }
`
```

So in short:

* Change `FIND_${singularConstantName}_BY_ID` to `Find${singularConstantName}ById` where `singularConstantName` should now be Pascal Case.

**Comment by thedavidprice** [comment]
@dthyresson could you help me triage this:
1. is it a v1-priority;
2. if so, is it critical for the current release or should we schedule it to a future release?

**Comment by dthyresson** [comment]
@thedavidprice It is not critical path in terms of v1 functionality, but the task is small enough, of low risk and provides consistent naming moving forward that I think it is worth doing.

**Comment by dthyresson** [comment]
Change is to use `singularPascalName` instead of `singularConstantName` and change the template surrounding operation name casing:

```
    -   query FIND_USER_PROFILE_BY_ID($id: Int!) {
    +   query FindUserProfileById(($id: Int!) {

    -   query FIND_POST_BY_ID($id: Int!) {
    +   query FindPostById($id: Int!) {
```
